test/index.js

```diff
@@ -0,0 +1,11 @@
+// Directions: https://github.com/webpack-contrib/istanbul-instrumenter-loader
```

Comment:
Can you rebase with master and pick up https://github.com/krakenjs/cross-domain-utils/commit/f31fd58da2aa9a9f22419e18e6221a370b9d1507 -- just tried this and it works just fine with the current test directory structure and imports/exports.